packages/core/core/src/types.js

```diff
@@ -71,6 +71,7 @@ export type Environment = {|
   isLibrary: boolean,
   minify: boolean,
   scopeHoist: boolean,
+  unsafeInlining: boolean,
```

Comment:
Now I'm wondering whether this should really be in the environment or whether the JS transformer should have its own config for this, kinda like how we did it for the bundler? Maybe we'll want to allow more detailed options for this in the future. Could be something like this, in package.json:

```json
{
  "@parcel/transformer-js": {
    "inlineFS": true,
    "inlineEnvironment": true | ["NODE_ENV", "MY_CUSTOM_VAR"]
  }
}
```

Not really sure when you'd need to configure that per target...

Reply:
> Not really sure when you'd need to configure that per target...

This is rather a security setting, not a configuration setting. So I think having a "global" option would be fine too